core/trie/trie_test.go

```diff
@@ -202,7 +202,7 @@ func TestPutZero(t *testing.T) {
 		t.Run("remove keys one by one, check roots", func(t *testing.T) {
 			var gotRoot *felt.Felt
 			// put zero in reverse order and check roots still match
-			for i := range 64 {
+			for i := range make([]int, 64) {
```

Comment:
I don't think this change is required either

Reply:
similar with previous, changes added because `golangci-lint` typecheck

Reply:
reverted to previous state